frontend/src/component/feature/FeatureStrategy/FeatureStrategySegment/RecentlyUsedSegments/RecentlyUsedSegments.tsx

```diff
@@ -0,0 +1,68 @@
+import { styled, Typography } from '@mui/material';
+import { useRecentlyUsedSegments } from './useRecentlyUsedSegments';
+import type { ISegment } from 'interfaces/segment';
+import { FeatureStrategySegmentChip } from '../FeatureStrategySegmentChip';
+import { useSegments } from 'hooks/api/getters/useSegments/useSegments';
+import { useUiFlag } from 'hooks/useUiFlag';
+
+type IRecentlyUsedSegmentsProps = {
+    setSegments?: React.Dispatch<React.SetStateAction<ISegment[]>>;
+};
+
+const StyledContainer = styled('div')(({ theme }) => ({
+    marginTop: theme.spacing(3),
```

Comment:
Wouldn't it be better to handle spacing around the container in the parent component? Feels like placing the component on the page is the responsibility of the caller and not the component itself. Also makes it easier if you want to evenly space items.

But yeah, not super important 💁🏼